packages/cli/src/utils/utils.ts

```diff
@@ -174,9 +174,11 @@ export function extractArrayValueFromTsManifest(content: string, key: string): s
   const [startIndex, endIndex] = findArrayIndicesTsManifest(content, key);
   return content.slice(startIndex, endIndex + 1);
 }
+// @ts-ignore
```

Comment:
Still to be fixed? This function looks like it would be better suited to being in generate-controller